letsencrypt/cli.py

```diff
@@ -723,10 +730,18 @@ def create_parser(plugins, args):
     return helpful.parser, helpful.args
 
 # For now unfortunately this constant just needs to match the code below;
-# there isn't an elegant way to autogenerate it in time.
-VERBS = ["run", "auth", "install", "revoke", "rollback", "config_changes",
-         "plugins", "--help"]
-
+# there isn't an elegant way to autogenerate it in time. pylint: disable=bad-whitespace
+VERBS = {
+    "run"            : run,
+    "auth"           : auth,
+    "install"        : install,
+    "revoke"         : revoke,
+    "rollback"       : rollback,
+    "config_changes" : config_changes,
+    "plugins"        : plugins_cmd
+}
+HELP_TOPICS = (["all", "security", "paths", "automation", "testing", "apache", "nginx"]
```

Comment:
Do you have to list all plugin names here manually? If so, then any new registered plugins (even those not controlled by us) will not be automatically registered here. In fact, you're missing `manual` and it's `--test-mode`.